memegen/routes/overview.py

```diff
@@ -0,0 +1,20 @@
+from flask import Blueprint, current_app as app, url_for, render_template
+# http://stackoverflow.com/questions/16061514/pylint-pylint-unable-to-import-flask-ext-wtf
+from flask.ext.api.decorators import set_renderers  # pylint: disable=E0611,F0401
+from flask.ext.api.renderers import HTMLRenderer  # pylint: disable=E0611,F0401
+
+
+blueprint = Blueprint('overview', __name__, url_prefix="/overview")
+
+
+def _urls_generator():
+    for template in sorted(app.template_service.all()):
+        url = url_for("image.get", key=template.key, path="hello-world",
```

Comment:
If you do `path = template.default or "hello/world"`, you can leverage the default text set in some of the templates.

See: https://github.com/jacebrowning/memegen/blob/3d7a21a1e04a5bee7f4ec8bca576b640b1087e92/memegen/routes/templates.py#L32-34